### DISCUSSION TITLE
Filterable frontpage (portfolio)

### DISCUSSION THREAD

**Discussion by tarjeieo** [Puzzlepart/prosjektportalen365#627] [Ideas]
Request from a PP365 customer
* Allow filters on the front page
* Filters needs to be URL-aware so that we can link to e.g. /frontpage.aspx?field1=Portfolio&field2=Eiendom
* Should filter the tiles on the frontpage
* Should be customizable by web part settings OR a backing configuration list
* Lookup to Prosjektkolonner-list to select filters

Input @okms @olemp @pzljanb @Remi749 ?

**Comment by pzljanb** [comment]
Filter on?
-Portfolio Front page
Filter should affect tiles, views +++
Project property (a new one?) could be used as parameter along with existing parameters like ProjectOwner?


-Project Front page
Filteres based on actual phase
Need for different content/ webparts based on phase?
Swop default page when changing phase. Pages can keep different content customized for each phase

**Comment by Remi749** [comment]
Should it also support setting up a predefined filtered front-page or just filtering the project-cards on the fly by users input (ex: FilterPanel). How about implementing sorting as well?
